hello-app/cloudshell/tutorial.md

```diff
@@ -0,0 +1,133 @@
+# Hello Application example
+
+## 
+
+This tutorial shows how to build and deploy a containerized Go web server application using [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine).
+
+You'll learn how to create a cluster, and how to deploy the application to the cluster so that it can be accessed by users.
+
+Let's get started!
+
+
+## Set your GCP project
+
+You can use an existing Google Cloud Platform project for this tutorial, or you can [create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+Open the <walkthrough-editor-spotlight spotlightId="menu-terminal">terminal</walkthrough-editor-spotlight> and follow the steps below:
+
+### 1. Set the default project    
+```bash
+gcloud config set project PROJECT_ID
```

Comment:
is there a way to have users set an env variable so that they can paste this command directly into cloud shell? 

eg, `gcloud config set project $PROJECT_ID`

Reply:
Done - I've added a step to create PROJECT_ID and COMPUTE_ZONE variables, then the subsequent gcloud config commands use the variables.